src/main/java/org/joshsim/lang/bridge/EngineBridge.java

```diff
@@ -167,12 +167,22 @@ public interface EngineBridge {
    */
   EngineValue getExternal(GeoKey key, String name, long step);
 
+  /**
+   * Get a configuration value by name, returning empty if not found.
+   *
+   * @param name The name of the configuration value to retrieve.
+   * @return Optional containing the configuration value, or empty if not found.
+   */
+  Optional<EngineValue> getConfigOptional(String name);
+
   /**
    * Get a configuration value by name.
    *
    * @param name The name of the configuration value to retrieve.
    * @return The configuration value as an EngineValue.
+   * @deprecated Use getConfigOptional instead
    */
+  @Deprecated
   EngineValue getConfig(String name);
```

Comment:
This is new enough (not even on `main` or `dev`). Let's remove instead of deprecating.